style.css

```diff
@@ -37,115 +41,120 @@ html {
 	padding: 15px;
 }
 
-.welcome-banner {
-	background-color: #EEEEEE;
-	height: 90vh;
-	
+@media screen and (max-width: 600px) {
+	.cv-button {
+	flex-direction: column;
+	font-size: 2rem;
+	}
 }
 
 .navigation-links {
 	color: #ffffff;
 	text-decoration: none;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
-    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #2866c9;
 }
 
-.hi-there-welcome-text {
-	display:flex;
-	align-items: center;
-	justify-content: center;
+@media screen and (max-width: 600px) {
+	.navigation-links {
+	font-size: 2rem;
+	}
+}
+
+/*navigation*/
+
+/*welcome*/
+
+.welcome-banner {
+	background-color: #EEEEEE;
+	height: 90vh;
+	flex-direction: column;
 	font-family: 'Varela Round', sans-serif;
-	font-size: 6rem;
-	padding-top: 10%;
 	color: #000000;
-	/*deciding whether to keep element*/
-/*    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #EFB7B7;*/
 }
 
-.welcome-text { 
+@media screen and (max-width: 600px) {
+	.welcome-banner {
+	flex-direction: column;
+	}
+}
+
+.small-title {
+	font-size: 6rem;
 	display:flex;
 	align-items: center;
 	justify-content: center;
-	font-family: 'Varela Round', sans-serif;
-	font-size: 11rem;
-	background-color: #EEEEEE;
-	margin: 0;
-	color: #000000;
-/*deciding whether to keep element*/
-/*	-webkit-text-stroke-width: 0.5px;
-	-webkit-text-stroke-color: #EFB7B7;*/
+	padding-top: 10%;
+	margin-bottom: 0;
 }
 
-.about-me-title {
+@media screen and (max-width: 600px) {
+	.small-title {
+	font-size: 4rem;
+	}
+}
+
+.sparkle-inline {
+	width: 10% ;
+	display: inline;
+}
+
+.large-title {
+	font-size: 11rem;
+	margin: 0;
 	display:flex;
 	align-items: center;
 	justify-content: center;
-	padding-top: 0;
-	font-family: 'Varela Round', sans-serif;
-	font-size: 3rem;
 }
 
-.scallop-mid-section {
+@media screen and (max-width: 600px) {
+	.large-title {
+	font-size: 7rem;
+	}
+}
+
+/*come back to this once looked at mobile formatting*/
+
+/*welcome*/
+
+/*about me*/
+
+.about-section {
 	background-color: #BD4B4B;
 	padding: 50px;
 	height: 100vh;
```

Comment:
Are you sure you want to use 100vh? If expanded to full screen on a large monitor its a lot of empty space, see image:
![image](https://user-images.githubusercontent.com/45392468/149964339-da9e0c91-783a-4efe-a160-51e5887f0bad.png)